webapp/components/wpt-results.html

```diff
@@ -242,16 +258,34 @@
             this.testFiles[testFileName][resultsURL] = testFiles[testFileName];
           });
         });
-
-        this.refreshDisplayedNodes();
       }
 
-      async fetchResults(resultsURL) {
+      async fetchResultsJSON(resultsURL) {
         const response = await window.fetch(resultsURL);
-        const testFiles = response.ok ? await response.json() : {};
+        let testFiles = {};
+        try {
+          testFiles = response.ok ? await response.json() : {};
```

Comment:
Sent https://github.com/web-platform-tests/wpt.fyi/pull/117 when I saw this line. Would it make sense to log non-OK statuses here too?

Reply:
Might be nice to retry or log, but not something I'd add this PR.